future-apps/payroll/contracts/Payroll.sol

```diff
@@ -600,11 +600,13 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
             timeDiff = timeDiff.add(1);
         }
 
-        uint64 lastPayrollDate = employee.lastPayroll.add(uint64(timeDiff));
```

Comment:
I was a little bit worried about this `uint64(timeDiff)` cast; although it's unlikely, at some point we may accidentally provide a `_paidAmount` that would cause `timeDiff` to be larger than MAX_UINT64.